data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1060 @@
+</div>
+
+<div type="textpart" subtype="section" xml:base="urn:cts:greekLit:tlg0018.tlg001.1st1K-eng1" n="4">
+<p>Accordingly no one, whether poet or historian, could ever give expression in an adequate manner to the beauty of his ideas respecting the creation of the world; for they surpass all the power of language, and amaze our hearing, being too great and venerable to be adapted to the senses of any created being.</p>
+</div>
+
+<div type="textpart" subtype="section" xml:base="urn:cts:greekLit:tlg0018.tlg001.1st1K-eng1" n="5">
+<p>That, however, is not a reason for our yielding to indolence on the subject, but rather from our affection for the Deity we ought to endeavour to exert ourselves even beyond our powers in describing them: not as having much, or indeed anything to say of
+<pb n="v.1.p.2"/>
+our own, but instead of much, just a little, such as it may be probable that human intellect may attain to, when wholly occupied with a love of and desire for wisdom.
+</p>
+</div>
+
+<div type="textpart" subtype="section" xml:base="urn:cts:greekLit:tlg0018.tlg001.1st1K-eng1" n="6">
+<p>For as the smallest seal receives imitations of things of colossal magnitude when engraved upon it, so perchance in some instances the exceeding beauty of the description of the creation of the world as recorded in the Law, overshadowing with its brilliancy the souls of those who happen to meet with it, will be delivered to a more concise record after these facts have been first premised which it would be improper to pass over in silence.
+</p>
+</div>
+
+<div type="textpart" subtype="section" xml:base="urn:cts:greekLit:tlg0018.tlg001.1st1K-eng1" n="7">
+<milestone unit="chapter" n="2"/>
+<p>For some men, admiring the world itself rather than the Creator of the world, have represented it as existing without any maker, and eternal; and as impiously as falsely have represented God as existing in a state of complete inactivity, while it would have been right on the other hand to marvel at the might of God as the creator and father of all, and to admire the world in a degree not exceeding the bounds of moderation.
+</p>
+</div>
+
+<div type="textpart" subtype="section" xml:base="urn:cts:greekLit:tlg0018.tlg001.1st1K-eng1" n="8">
+<p>But Moses, who had early reached the very summits of philosophy, <note xml:lang="eng" n="2.1">This is in accordance with the description of him in the Bible, where he is represented as being learned in all the wisdom of the Egyptians. </note> and who had learnt from the oracles of God the most numerous and important of the principles of nature, was well aware that it is indispensable that in all existing things there must be an active cause, and a passive subject; and that the active cause is the intellect of the universe, thoroughly unadulterated and thoroughly unmixed, superior to virtue and superior to science, superior even to abstract good or abstract beauty;</p>
```

Comment:
- notes are not numbered when they are inline. (Good idea to indicate the placement here).

- plus, this is note 2.1 as is the next one, so the numbering is both not needed and duplicated

- lang attribute not needed.